pkg/controller/nodes/task/transformer.go

```diff
@@ -50,7 +50,8 @@ func trimErrorMessage(original string, maxLength int) string {
 	return original[0:maxLength/2] + original[len(original)-maxLength/2:]
 }
 
-func ToTaskExecutionEvent(taskExecID *core.TaskExecutionIdentifier, in io.InputFilePaths, out io.OutputFilePaths, info pluginCore.PhaseInfo) (*event.TaskExecutionEvent, error) {
+func ToTaskExecutionEvent(taskExecID *core.TaskExecutionIdentifier, in io.InputFilePaths, out io.OutputFilePaths, info pluginCore.PhaseInfo,
```

Comment:
is interruptible entirely decided by node? I thought it is decided by the task execution itself?

Reply:
it can be decided by the task, see here: https://github.com/lyft/flyteidl/blob/8aff8a021c0127c273f03e0311152c349972ee46/protos/flyteidl/core/tasks.proto#L82 but is stored in the node metadata

Reply:
okay wow looking at the code though, I only see it ever initialized from the workflow-level setting https://github.com/lyft/flytepropeller/blob/master/pkg/controller/nodes/node_exec_context.go#L206,L209 😮 

this should be fixed

Reply:
ohh are you saying that the node will change this setting lets say for retries etc?

Reply:
to close the loop on this:

compiler _does_ set the interruptible bit for nodes https://github.com/lyft/flytepropeller/blob/master/pkg/compiler/transformers/k8s/node.go#L60,L68

which is indeed set correctly from the task def https://github.com/lyft/flytekit/blob/master/flytekit/common/tasks/task.py#L137